http/src/main/java/io/hyperfoil/http/HttpUtil.java

```diff
@@ -11,13 +12,25 @@
 import io.hyperfoil.impl.Util;
 import io.netty.buffer.ByteBuf;
 import io.netty.util.AsciiString;
+import io.netty.util.concurrent.FastThreadLocal;
 
 public final class HttpUtil {
+   private static final String UTC_STR = "UTC";
```

Comment:
Use AsciiString here if the seq tends to be an AsciiString

Reply:
Done, even if I adon't think there is much impact

Reply:
Only if are the most used there 🙏 you can verify by debugging